src/main/java/frc/robot/subsystems/ElevatorSubsystem.java

```diff
@@ -234,6 +241,20 @@ public Command setLevel(double pos) {
         .withName("setLevel" + pos);
   }
 
+  public Command setBrakeMode() {
+    return runOnce(
+        () -> {
+          m_motor.setControl(new StaticBrake());
+        });
+  }
+
+  public Command setCoastMode() {
+    return runOnce(
+        () -> {
+          m_motor.setControl(new CoastOut());
+        });
+  }
```

Comment:
```suggestion
  public Command holdCoastMode() {
    return startEnd(
          () -> {
            m_motor.setControl(new CoastOut());
          },
          () -> {
            m_motor.setControl(new StaticBrake());
          })
        .ignoringDisable(true)
        .withName("Hold elevator coast");
  }
```
By putting the coast and brake into the same command, we can guarantee we can't set coast without setting brake when we're done. This also moves more of the logic (ignoring disable and naming) into the command factory so that we won't need to duplicate that if we use this somewhere else (e.g., bind to a Trigger that represents a physical button on the robot).